.semaphore/live-site-deploy.yml

```diff
@@ -20,5 +20,5 @@ blocks:
             - npm install
             - gem install bundler
             - bundle install
-            - JEKYLL_ENV=production bundle exec jekyll build
-            - aws s3 cp --recursive ./_site s3://kafka-tutorials.confluent.io/
+            - JEKYLL_ENV=production bundle exec jekyll build --baseurl /tutorials
```

Comment:
Should the `pr-staging-deploy.yml` and `staging-site-deploy.yml` files get updated as well?